scss/civicrm/common/_base.scss

```diff
@@ -315,3 +315,23 @@ a.sorting_asc::before {
 a.sorting_desc::before {
   @include fa-icon($font-size-base, $fa-var-sort-desc, $brand-primary);
 }
+
+#alpha-filter {
+  li {
+    color: $crm-gray-matte;
+    font-family: $font-family-base;
+    font-size: $font-size-base;
+    font-weight: $crm-font-weight-h3;
+
+    a,
+    a:link {
```

Comment:
why is `a:link` important to specify?

Reply:
That's due to core CSS file it overrides :link color over normal a tag and we need to prevent that.